src/components/Input.js

```diff
@@ -137,15 +137,13 @@ export const Input = props => {
       useNativeDriver: false,
     }).start();
 
-    {
-      !!label &&
-        inputRef.current &&
-        inputRef.current.setNativeProps({
-          style: {
-            top: isFocused ? 10 : 0,
-          },
-        });
-    }
+    !!label &&
+      inputRef.current &&
+      inputRef.current.setNativeProps({
```

Comment:
@Amaljith-K Here line number 141 and 142 can be written as `inputRef.current?.setNativeProps` with null coalescing operator. Can this be a useful lint rule?

Reply:
We cannot enforce that via rule because they are not equivalent.
If `str` is an empty string, 
`str && str.at(0)` returns an empty string
`str?.at(0)` returns `undefined`

Since ESLint has no way of knowing types of variables in JS, we cannot enforce that rule.